### ISSUE TITLE
Enable creation of peg-in and peg-out-request burnchain operations

### ISSUE THREAD

**Issue by netrome** [Trust-Machines/core-eng#406]

**Comment by netrome** [comment]
I just had a meeting with @radicleart. He intends to support us in deploying an alpha version of the sBTC bridge. Notes from our meeting: [https://docs.google.com/document/d/1RNBY31wJNbCyw5Pmnoczng9uIgZ3tVbPa7ZFZL355RE/](https://www.google.com/url?q=https://docs.google.com/document/d/1RNBY31wJNbCyw5Pmnoczng9uIgZ3tVbPa7ZFZL355RE/&sa=D&source=calendar&ust=1686141953566120&usg=AOvVaw2YeYciYMtAJEbumsjfUGBM)

**Comment by netrome** [comment]
This is done as our sbtc-op CLI was just merged.

**Comment by igorsyl** [comment]
Rename to sbtc-cli?

**Comment by netrome** [comment]
> Rename to sbtc-cli?

Makes sense. I created a ticket to do so: https://github.com/Trust-Machines/core-eng/issues/460
